__tests__/app.test.js

```diff
@@ -26,6 +26,44 @@ describe('/api/topics', () => {
     })
 });
 
+describe('/api/articles/:article_id', () => {
+    test('GET 200: returns the article with the article_id given as a parameter in the URL', () => {
+        return request(app)
+        .get('/api/articles/1')
+        .expect(200)
+        .then((response) => {
+            const {article} = response.body
```

Comment:
We also want to check our article_id is 1, to ensure we have the correct article back